presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraTestingUtils.java

```diff
@@ -220,6 +222,24 @@ public static void createTableAllTypesPartitionKey(CassandraSession session, Sch
         insertTestData(session, table, date, 9);
     }
 
+    public static void createTableTupleType(CassandraSession session, SchemaTableName table)
+    {
+        String typeName = "type_tuple_type";
+
+        session.execute("DROP TABLE IF EXISTS " + table);
+
+        session.execute("CREATE TABLE " + table + " (" +
+                " typeinteger int PRIMARY KEY, " +
+                " typetuple frozen<tuple<int, int>>" +
```

Comment:
Let's rename the primary key column to `key` as other test methods use.
How about adding more type to tuple? eg. `tuple<int, text, float>`